docs/internals/protocol/overview.rst

```diff
@@ -52,6 +52,10 @@ The following data types are used in the descriptions:
                int16  key;
                bytes  value;
            };
+    * - ``uuid``
+      - a simple array of 16 bytes with no length prefix, equivalent to
+        `byte[16]`
+
```

Comment:
I think you need to shift this new block one or two spaces to the right. Align with line 41

Reply:
I've fixed indentation earlier. This is just single backticks against double backticks that driving me crazy when switching between rst and md :)

Reply:
Yeah :)  We added a special test for that because we were so tired of hunting single backticks down.